src/test/scala/uk/gov/hmrc/test/ui/pages/section5/ProcedureCodesPage.scala

```diff
@@ -22,31 +22,30 @@ import uk.gov.hmrc.test.ui.pages.base.TariffLinks.{itemsProcedureCodes, itemsPro
 import uk.gov.hmrc.test.ui.pages.base.{BasePage, Detail, Details}
 import uk.gov.hmrc.test.ui.pages.section1.DeclarationChoicePage.isClearance
 import uk.gov.hmrc.test.ui.pages.section2.DetailKeys.EntryIntoDeclarantsRecords
-import uk.gov.hmrc.test.ui.pages.section5.DetailsKeys.{ItemIds, ProcedureCode}
+import uk.gov.hmrc.test.ui.pages.section5.DetailsKeys.{ItemIds, ProcedureCode, ProcedureCodeLabel}
 
 object ProcedureCodesPage extends BasePage {
 
   val pageId = "procedure-codes"
   def backButtonHref: String = DeclarationItemsListPage.path
-  def path: String           = itemUrl("procedure-codes")
-  val title: String          = "What is the procedure code for this item?"
+  def path: String = itemUrl("procedure-codes")
+  val title: String = "What is the procedure code for this item?"
 
-  override val expanderHrefs: Map[String, Seq[String]] = Map(
-    Common    -> List(itemsProcedureCodes),
-    Clearance -> List(itemsProcedureCodesCL)
-  )
+  override val expanderHrefs: Map[String, Seq[String]] =
+    Map(Common -> List(itemsProcedureCodes), Clearance -> List(itemsProcedureCodesCL))
 
   override def pageLinkHrefs: Seq[String] =
     if (isClearance && detail(EntryIntoDeclarantsRecords) == yes) super.pageLinkHrefs
```

Comment:
`isEidr` can be imported from `EntryIntoDeclarantRecordsPage.scala`. Would you please then replace this line with
```
if (isEidr) super.pageLinkHrefs
else super.pageLinkHrefs ++ List(
  ...
```